frontend/app/src/App.tsx

```diff
@@ -137,6 +137,7 @@ interface State {
   connectionState: ConnectionState
   elements: AppRoot
   isFullScreen: boolean
+  hideFullScreenButtons: boolean
```

Comment:
?: Do we need `hideFullScreenButtons` in state since it is always false for our purposes?

Reply:
Hm I think it's good to explicitly define it. What do you think?

Reply:
I can see cases for both. If you have a strong preference, i'll be happy to change it!

Reply:
I just think its confusing since we don't have a mechanism in place that actually updates it
Thoughts @kmcgrady?

Reply:
Hm I was thinking it's good to define it explicitly since Ken was afraid that it would be changed somehow. However, it's probably better to just set it to what `LibContext` sets it to so I'll remove it. Everything else look good?